worker/src/test/resources/application.properties

```diff
@@ -64,4 +61,5 @@ optout.used=${AB2D_OPT_OUT_ENABLED:#{false}}
 
 ## ------------------------------------------------------------------------------ Earliest Data Date
 bfd.earliest.data.data=01/01/2020
-
+bfd.earliest.data.date.special.contracts=01/01/1950
+bfd.special.contracts=S0001,Z0001
```

Comment:
Since this is referenced in code do you need to add it to the regular properties file too?